__tests__/commands/global.js

```diff
@@ -22,31 +22,31 @@ async function runGlobal(
   name: string,
   checkGlobal?: ?(config: Config, reporter: Reporter, out: string) => ?Promise<void>,
 ): Promise<void> {
+  let out = '';
+  const stdout = new stream.Writable({
```

Comment:
This is copied and pasted in a few places, what about making a `mockOutputStream` module to encapsulate it?